ts/nni_manager/core/nniDataStore.ts

```diff
@@ -67,7 +67,7 @@ class NNIDataStore implements DataStore {
         return this.db.queryLatestExperimentProfile(experimentId);
     }
 
-    public storeTrialJobEvent(
+    public async storeTrialJobEvent(
```

Comment:
If it's made async, try-catch should be preferred over promise.catch()

Reply:
let me remove `async`.